Gruntfile.js

```diff
@@ -69,6 +70,18 @@ module.exports = exports = function(grunt) {
                         // just to add the BANNER
                         // without adding an extra grunt module
                         'dist/localforage.nopromises.js'
+                    ],
+                    // Node output files
+                    'dist/localforage.node.js': [
+                        'src/utils/node_banner.js',
+                        'bower_components/es6-promise/promise.js',
+                        'dist/localforage.nopromises.js'
+                    ],
+                    'dist/localforage.nopromises.node.js': [
+                        // just to add the BANNER
+                        // without adding an extra grunt module
+                        'src/utils/node_banner.js',
+                        'dist/localforage.nopromises.js'
```

Comment:
[I did that](https://github.com/mozilla/localForage/pull/551/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R55), and I agree that the `nopromises` version may be a bit silly for node.